### PR TITLE
Church boogaloo

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Nerfs musket to not be literally better lightfall.
Nerfs cost on the new church hardsuit.
Renames the tactical absolutist belt to absolutist tactical belt to be in line with the other one.
Rearranges the church disks and removes the non church related things (EMT belts, wallets etc).
Fixes eternal candle description.
Fixed naming on firemodes for the cleanser carbine.
Removed medkit from EOTP due to it being redudant to have it there.
Adds path related armors (same stats as vector) that you get from an armor bundle within lockers and numerical clothes, all sprites made by Polyushko just like last time.

<hr>
</details>

## Changelog
:cl:
add: Path based armors and numerical sprites
tweak: Church Disks
balance: Church hardsuit and musket
fix: Eternal Candle description
/:cl:

### CODE REVIEW

**Comment by r4d6** [comment]
What about the already-existing Vector armor alt appearances?

File: code/modules/core_implant/cruciform/machinery/armaments.dm [hunk 1]
```diff
@@ -236,9 +236,3 @@
 	cost = 50
 	min_cost = 25
 	path = /obj/item/computer_hardware/hard_drive/portable/design/nt/new_guns
-
-/datum/armament/item/gun/firstaid
```
Comment (Trilbyspaceclone) [inline code/modules/core_implant/cruciform/machinery/armaments.dm:240]: Why removing thiss?

File: code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm [hunk 2]
```diff
@@ -54,85 +52,14 @@
...
 	)
 
-
 /obj/item/computer_hardware/hard_drive/portable/design/nt_bioprinter
 	disk_name = "Church of Absolute Bioprinter Products and Utilities"
 	desc = "This disk is sole property of the church, the files within are encrypted and should not be used, taken, or tested by anyone not affiliated with the Absolute."
 	icon_state = "neotheology_testament_u"
-
 	license = 20
-	designs = list(
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm:80]: Did you mean to make this dis have 0 designs?

File: code/modules/core_implant/cruciform/machinery/armaments.dm (hunk 1 above)
Comment (Lamasmaster) [reply code/modules/core_implant/cruciform/machinery/armaments.dm:240]: It's a noob trap, there isn't ever a reason to order a medkit that is almost same cost as something like a purger.

File: code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm (hunk 2 above)
Comment (Lamasmaster) [reply code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm:80]: Shouldn't it inherit designs from the base disk just have license costs ?

File: code/modules/core_implant/cruciform/machinery/armaments.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/core_implant/cruciform/machinery/armaments.dm:240]: Fair

File: code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm:80]: What base designs?

Comment (Lamasmaster) [reply code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm:80]: The pathing was all weird so now it should be good

**Comment by Lamasmaster** [comment]
> What about the already-existing Vector armor alt appearances?

They stay on the old vector armor these are new ones

File: code/datums/autolathe/biomatter.dm [hunk 3]
```diff
@@ -189,6 +193,7 @@
 
 /datum/design/bioprinter/nt_clothes/knight_rig
 	name = "Knight Hardsuit Module"
+	materials = list(MATERIAL_BIOMATTER = 50, MATERIAL_GOLD = 5, MATERIAL_SILVER = 5, MATERIAL_PLASTEEL = 10)
```
Comment (ThePainkiller) [inline code/datums/autolathe/biomatter.dm:196]: **VERY** appreciated, I got complains from church players that the armor would've been _"too good"_ for its values and utility, and it only took metal and plastic to build, this selection of materials is pretty fair and balanced.

File: code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm:80]: Looks good

**Pull request merged**